src/styles/main.scss

```diff
@@ -10,17 +10,78 @@ body {
   font-weight: 900;
 }
 
+.move {
+  &-up {
+    &--1 {
+      transform: translateY(-85px);
+    }
+    &--2 {
+      transform: translateY(-170px);
+    }
+    &--3 {
+      transform: translateY(-255px);
+    }
+  }
+
+  &-down {
+    &--1 {
+      transform: translateY(85px);
+    }
+    &--2 {
+      transform: translateY(170px);
+    }
+    &--3 {
+      transform: translateY(255px);
+    }
+  }
+
+  &-left {
+    &--1 {
+      transform: translateX(-85px);
+    }
+    &--2 {
+      transform: translateX(-170px);
+    }
+    &--3 {
+      transform: translateX(-255px);
+    }
+  }
+
+  &-right {
+    &--1 {
+      transform: translateX(85px);
+    }
+    &--2 {
+      transform: translateX(170px);
+    }
+    &--3 {
+      transform: translateX(255px);
+    }
+  }
```

Comment:
It's strange that you've prepared those styles but aren't using them, yet the transition is still working.

Reply:
The code responsible for animations is commented out because it causes the interruption of automated tests.

Reply:
But apart from that, there is no difference in the code on the website.

Reply:
If it would be commented out I would be able to see that part.